device/arista/x86_64-arista_7060x6_16pe_384c/Arista-7060X6-16PE-384C-O128S2-COPPER-LAB/sai.profile

```diff
@@ -0,0 +1 @@
+SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/th5-a7060x6-16pe-384c.config.bcm
```

Comment:
@Staphylo could you check flex counter block is included in all the BCM file like https://github.com/sonic-net/sonic-buildimage/pull/21362/files

Reply:
I will check and create a PR accordingly.

Reply:
@kperumalbfn the changes from the PR you linked are actually included in the 3 bcm files of this PR.
I believe we should go forward and cherry-pick this change to 202411